api/app/Enums/AssessmentDecision.php

```diff
@@ -4,7 +4,7 @@
 
 enum AssessmentDecision
 {
-    case NOT_SURE;
```

Comment:
Could you update api/database/factories/AssessmentResultFactory.php to sometimes seed nulls?

Reply:
Good call! Added in [add null decision to factory](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8955/commits/0d94c66619381d7cedd5a6c9618fb47ebbe2e12a)

Also, I didn't realize we added a test that used the old value, should build now after [fix tracker tests after merge](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8955/commits/7e962fe9ed931ffe1657ebf808dab461b62735b1)